common-custom-user-data-gradle-plugin/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
+- Fully support the [Gradle configuration cache](https://docs.gradle.org/current/userguide/configuration_cache.html)
+- Add the ability to overridie various Gradle Enterprise configuration settings via system properties
```

Comment:
1. Spelling "override"
2. Shorten this to: "Allow the override of Gradle Enterprise configuration settings via system properties"

Reply:
Done!